### ISSUE TITLE
Pilot the accessibility survey

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-team#481]
The accessibility survey should be piloted before distributed to work out any issues.

- [ ] Identify participants for pilot and reach out to invite them to pilot the survey
- [ ] Obtain feedback from pilot survey participants
- [ ] Incorporate any changes needed to the survey based on feedback
